pkg/service/pipelinerun.go

```diff
@@ -100,16 +100,16 @@ func (s *service) ListPipelineRuns(ctx context.Context, req *pipelinepb.ListPipe
 
 	var referenceIDs []string
 	for _, pipelineRun := range pipelineRuns {
-		if canViewPrivateData(pipelineRun.RequesterUID, requesterUID) {
+		if canViewPrivateData(pipelineRun.RunnerUID, userUID) {
```

Comment:
So this means that the run data will be visible when the authenticated user is the same that triggered the endpoint?

Do we want org runs to be visible for all the users within an organisation? I think that's why we had the requester UID check instead of the user UID